### PR TITLE
Add Zagreb 2018 TMS

### PR DESCRIPTION
**Pull request by hbogner**
Add Zagreb 2018 TMS layer to index

### CODE REVIEW

File: sources/europe/hr/osm-hr-zagreb-2018.geojson [hunk 1]
```diff
@@ -0,0 +1,1288 @@
...
+    "type": "Feature",
+    "properties": {
+        "id": "osm-hr-zagreb-2018",
+        "name": "osm-hr: Zagreb 2018 Aerial imagery",
+        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/hr/osm-hr.png",
+        "start_date": "2018",
+        "end_date": "2018",
+        "url": "https://tms.osm-hr.org/zagreb-2018/{zoom}/{x}/{-y}.png",
+        "permission_osm": "explicit",
+        "license_url": "https://narodne-novine.nn.hr/clanci/sluzbeni/2013_02_25_403.html",
```
Comment (andrewharvey) [inline sources/europe/hr/osm-hr-zagreb-2018.geojson:11]: Is this akin to the data being the the public domain?

Comment (hbogner) [reply sources/europe/hr/osm-hr-zagreb-2018.geojson:11]: It's the same license as https://github.com/osmlab/editor-layer-index/pull/663 and https://github.com/osmlab/editor-layer-index/pull/643.

Comment (andrewharvey) [reply sources/europe/hr/osm-hr-zagreb-2018.geojson:11]: Oh yes we discussed this in #643, thanks. http://data.gov.hr/open-licence-republic-croatia seems clear to me, except it does require attribution for use so I'd suggest adding the `attribution` field https://github.com/osmlab/editor-layer-index/blob/8838470547becaf52c9d387585813fb66912bb16/schema.json#L145-L159 to all these layers under that license.

Comment (hbogner) [reply sources/europe/hr/osm-hr-zagreb-2018.geojson:11]: I added attribution to https://wiki.openstreetmap.org/wiki/Contributors#Croatia and they said it's ok. I can also add attribution field to geojson if you think it's needed.
Also remake of https://wiki.openstreetmap.org/wiki/WikiProject_Croatia/Podloge with proper up to date info is planed

Comment (andrewharvey) [reply sources/europe/hr/osm-hr-zagreb-2018.geojson:11]: I don't think it hurts to also add the attribution here per the schema.

Comment (hbogner) [reply sources/europe/hr/osm-hr-zagreb-2018.geojson:11]: Done

**Review by andrewharvey — APPROVED** [review]

**Comment by tordans** [comment]
Hi! Would this new layer be the "best" layer for the region? My understand is, that the recommended/best layer can/should have the `best: true` attribute so an editor knows to pic this as the default.

Example: https://github.com/osmlab/editor-layer-index/blob/00a27c260834035d1cd2d18fe2ae9ffeefb78b8c/sources/europe/de/Berlinaerialphotograph2019.geojson?short_path=acee593#L11

**Comment by hbogner** [comment]
Yes, this would be the best possible resolution and position layer for the region, also it's the latest imagery from all the available state imagery.

**Comment by simonpoole** [comment]
@hbogner any chance to get a privacy_policy_url for the layer?

**Comment by hbogner** [comment]
> @hbogner any chance to get a privacy_policy_url for the layer?

In short: No

Explained: https://geoportal.zagreb.hr/ has no privacy policy online. Also most of state administration is not so acquainted with licenses and legal rights to reuse data.
I explained it in Knin pull request: https://github.com/osmlab/editor-layer-index/pull/643#issuecomment-493381895

I requested the data according to that law and I received the data.

I'm planing to edit https://wiki.openstreetmap.org/wiki/WikiProject_Croatia/Podloge with specifications about those layers, and quotes from official documents that granted me that data. Can that page can be used as privacy_policy_url ?

**Comment by hbogner** [comment]
Is this OK?
https://tms.osm-hr.org/privacy.htm

**Comment by simonpoole** [comment]
> 
> 
> Is this OK?
> https://tms.osm-hr.org/privacy.htm

It's a bit minimal :-) But if the "we" is identified, better than nothing.

**Comment by hbogner** [comment]
It's minimal because we track nothing but apache/nginx logs by default.
I defined who "we" are :)
Better?

**Comment by simonpoole** [comment]
That's Ok.

**Comment by hbogner** [comment]
Then @grischard can merge now :)

**Comment by Klumbumbus** [comment]
A little part in the north west is missing in the polygon, just in case you want to fix it.

**Comment by hbogner** [comment]
Fixed, will merge in next pull request

**Pull request merged**